js/apps/admin-ui/src/realm-settings/TokensTab.tsx

```diff
@@ -85,6 +88,13 @@ export const RealmSettingsTokensTab = ({
     defaultValue: false,
   });
 
+  const strategy: string = (useWatch({
+    control,
+    name: convertAttributeNameToForm("attributes.oid4vci.time.claims.strategy"),
+  }) ??
+    realm.attributes?.["oid4vci.time.claims.strategy"] ??
+    "off") as string;
```

Comment:
It's a bit odd whenever we have to use things like "as string". What about leveraging the defaultValue mechanism, as seen in other places.


```suggestion
  const strategy: string = useWatch({
    control,
    name: convertAttributeNameToForm("attributes.oid4vci.time.claims.strategy"),
    defaultValue: "off"
  });
```